src/types/socket.ts

```diff
@@ -5,8 +5,11 @@ import { Socket } from 'socket.io-client';
 import { Empty } from 'types';
 import { Action, AsyncAction } from 'types/state';
 
-export type ClientEvents = 'join_game' | 'leave_game' | 'join_auth' | 'leave_auth';
-export type ServerEvents = 'start_game' | 'new_odds' | 'new_move' | 'game_over' | 'wager_result' | 'pool_wager' | 'socket_error' | 'game_error';
+export type ClientEvents =
+    'join_game' | 'leave_game' | 'join_auth' | 'leave_auth' | 'pool_wager' | 'game_chat';
+export type ServerEvents =
+    'start_game' | 'new_odds' | 'new_move' | 'game_over' | 'wager_result' | 'pool_wager' |
+    'game_chat' | 'socket_error' | 'game_error';
```

Comment:
this doesn't handle the filtering for swear words yet, right?

Reply:
The filter is in the backend.